jobs/config.json

```diff
@@ -62,7 +62,9 @@
   },
   "ci-kubernetes-bazel-test": {
     "args": [
-      "--test=--test_tag_filters=-integration --flaky_test_attempts=3 //cmd/... //pkg/... //federation/... //plugin/... //third_party/... //hack/... //hack:verify-all //vendor/k8s.io/..."
+      "--test=//cmd/... //pkg/... //federation/... //plugin/... //third_party/... //hack/... //hack:verify-all //vendor/k8s.io/...",
+      "--test-args=--test_tag_filters=-integration",
```

Comment:
can you only give a single test-arg at a time? or could you do
`"--test-args=--test_tag_filters=-integration --flaky_test_attempts=3"`?